palantir-java-format/src/test/resources/com/palantir/javaformat/java/testdata/CL240367479.input

```diff
@@ -1,4 +1,4 @@
-package foo;;
+package foo;
```

Comment:
Why was the test **input** changed?

Reply:
";;" is not valid on java 21.
"Disallow Extra Semicolons Between "import" Statements"
https://www.oracle.com/java/technologies/javase/21-relnote-issues.html#JDK-8027682

I was not able to disable the test only for Java21